aries-site/src/pages/templates/empty-state.mdx

```diff
@@ -0,0 +1,130 @@
+import { Example } from '../../layouts';
+import {
+  ActionEmptyState,
+  CardEmptyState,
+  EmptyStateAnatomy,
+  EmptyStateExample,
+  ErrorManagementEmptyState,
+  LayerEmptyState,
+  PageEmptyState,
+  NoData,
+} from '../../examples';
+import { Checkmark } from 'grommet-icons';
+
+<Example code="#" figma="#">
+  <EmptyStateExample />
+</Example>
+
+## When to use
+
+Common empty state use cases include:
+
+- “Day 0” use cases when a user first starts using an application
+- When the user completes an action that returns no results
+- When an error prevents the system from returning any data
+
+It is our responsibility as designers to provide our users with clear, concise guidance on why the data isn’t present and how to move forward in their journey.
+
+A well-designed empty state can improve the user experience and reduce confusion by setting clear expectations for what the user should do next.
+
+Here are a few characteristics that make a well-designed empty state:
+
+1.  **Clear messaging:** The empty state should have simple and clear messaging that tells the user what information is missing, why it is missing, and what they can do next to access the information. Follow the voice and tone guidance within the design system to remain clear and empathetic to the user.
+1.  **Visual cues:** The empty state should use iconography to help convey the message and create a more engaging user experience. The icon should be relevant to the context of the interface and help the user understand why it's empty.
```

Comment:
Wondering if we should just provide a handful of icons that folks should pick from. Realistically, there are only a few I could think of that would be used for empty states.

Reply:
If you have a shortlist, we can add. I think for the first pass of the page it's sufficient to say the icon should be relevant to the context. I'm not sure we want to be in the business of trying to cover every context that would have an empty state.

Reply:
Agree with avoiding trying to over-engineer. If we do have a short-list in mind, we could add a simple sentence saying "Some more common icons are ...." That said, I think cases where the icon may be related to the type of objects in a data collection (e.g. access points, servers, clusters, etc.) so leaving it as "relevant to the context" may be the best way to go for now.

Reply:
Looking at the use cases more, I agree with keeping them "relevant to the context." Originally thought it was mostly just status indicators, the error, close, and a few other misc. icons.